lib/msal-react/test/components/AuthenticatedTemplate.spec.tsx

```diff
@@ -26,8 +26,9 @@ describe("AuthenticatedTemplate tests", () => {
         },
     };
 
-    beforeEach(() => {
+    beforeEach(async () => {
         pca = new PublicClientApplication(msalConfig);
+        await pca.initialize();
```

Comment:
It's not clear to me why this is needed in each of these test files. msal-react calls this under the hood and these tests pass today

Reply:
You won't have the standardcontroller until initialize is called.... in this way if you don't need it... we don't have to download it....